### PR TITLE
feat: change navPrimary to fit ux requirements

### PR DESCRIPTION
**Pull request by matheusgomes062**
Connected to [APPS-1634](https://jira.library.ucla.edu/browse/APPS-1634)

image

image

**Notes:**

Just changed the component, there was no need to create a new one.

**Checklist:**

-   [x] I checked that it is working locally in the dev server
-   [x] I checked that it is working locally in the storybook
-   [x] I checked that it is working locally in the 
library-website-nuxt dev server
-   [x] I added a screenshot of it working
-   [x] UX has reviewed and approved this
-   [ ] I assigned this PR to someone on the dev team to review
-   [x] I used a conventional commit message
-   [x] I assigned myself to this PR

### CODE REVIEW

File: src/lib-components/NavPrimary.vue [hunk 1]
```diff
@@ -172,6 +181,12 @@ export default {
         width: auto;
     }
 
+    .title {
+        @include step-1;
+        color: var(--color-primary-blue-03);
+        text-transform: inital;
```
Comment (axamei) [inline src/lib-components/NavPrimary.vue:187]: typo: should be `initial`

Comment (dianneweinthal) [reply src/lib-components/NavPrimary.vue]: add property `letter-spacing: normal; `

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**